README.md

```diff
@@ -1,12 +1,12 @@
 # LiMeApp
-[![Greenkeeper badge](https://badges.greenkeeper.io/libremesh/lime-app.svg)](https://greenkeeper.io/) [![Build Status](https://travis-ci.org/libremesh/lime-app.svg?branch=develop)](https://travis-ci.org/libremesh/lime-app) [![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
```

Comment:
Why would we stop using greenkeeper?
I think it is one of those things that are super relevant if we regularly use it, or a reminder of how delayed we are in keeping dependencies out of date...

Reply:
You are right! I went too fast here...
Greenkeeper has been "closed" and they recommend to move to "Snyk".
I've open an issue to do this:  #323 and revert the badge removal